tests/core/tests/authentication.py

```diff
@@ -33,8 +32,8 @@ def test_is_authenticated(self):
         auth = Authentication()
         request = HttpRequest()
         # Doesn't matter. Always true.
-        self.assertTrue(auth.is_authenticated(None))
-        self.assertTrue(auth.is_authenticated(request))
+        self.assertEqual(auth.is_authenticated(None), True)
```

Comment:
auth.is_authenticated() sometimes returns HttpUnauthorized, which is a truthy value for assertTrue.  We should be very careful about using assertTrue for anything that doesn't return True/False.

We should probably consider fixing is_authenticated() because any is_* method should return only True or False, but it could break existing APIs.